apps/challenges/migrations/0065_add_cluster_details_to_challengeevaluationcluster_model.py

```diff
@@ -0,0 +1,31 @@
+# -*- coding: utf-8 -*-
```

Comment:
Can we please change this file name to - `0065_add_cluster_details_to_challengeevaluationcluster_model.py ` -> `0065_add_cluster_details_to_challenge_evaluation_cluster_model.py `